### ISSUE TITLE
Builds are broken

### ISSUE THREAD

**Issue by johnmarkwayve** [grailbio/bazel-toolchain#7]
It appears that using these toolchains is now broken, the issue seems to be related to https://github.com/bazelbuild/bazel/issues/6665

Essentially, when I try to build almost anything, I get the following error:

```
$ bazel build //basically:anything
ERROR: /home/user/.cache/bazel/_bazel_user/b66616ba40524aac761ec07c06bbfdfc/external/local_config_cc/BUILD:57:1: in cc_toolchain rule @local_config_cc//:cc-compiler-k8: Error while selecting cc_toolchain: Toolchain identifier 'local' was not found, valid identifiers are [clang-linux, clang-darwin]
ERROR: Analysis of target '//core/external/ros2:ros2' failed; build aborted: Analysis of target '@local_config_cc//:cc-compiler-k8' failed; build aborted
INFO: Elapsed time: 0.111s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)
```

See also https://github.com/bazelbuild/bazel-toolchains/issues/236

**Comment by siddharthab** [comment]
Should be fixed by 98d700d9d1738f69fb5b5bb95e61adce993cf083.

The proper fix in bazel is https://github.com/bazelbuild/bazel/commit/683c302129b66a8999f986be5ae7e642707e978c.

**Comment by johnmarkwayve** [comment]
Thanks for the fix. Unfortunately, this now doesn't work with bazel 0.19.0. To get it working I had to reintroduce

```
default_toolchain {
  cpu: "k8",
  toolchain_identifier: "clang-linux"
}
```

to the `CROSSTOOL.tpl` file
